setup.sh

```diff
@@ -37,7 +37,11 @@ function updateenv () {
 function install_talib () {
     curl -O -L http://prdownloads.sourceforge.net/ta-lib/ta-lib-0.4.0-src.tar.gz
```

Comment:
we do have this file in our repository ... so we could ommit downloading it again (which can lead to failures close to month-end) - and should speed up the installation process.

Especially since the file extracted will be the one in the repo - the one downloaded will replace ours.